readthedocs/settings/base.py

```diff
@@ -177,8 +189,22 @@ def INSTALLED_APPS(self):  # noqa
             apps.append('readthedocsext.donate')
             apps.append('readthedocsext.embed')
             apps.append('readthedocsext.spamfighting')
+        if self.RTD_EXT_THEME_ENABLED:
+            apps.append('readthedocsext.theme')
         return apps
 
+    @property
+    def CRISPY_TEMPLATE_PACK(self):
+        if self.RTD_EXT_THEME_ENABLED:
+            return 'semantic-ui'
+        return None
+
+    @property
+    def CRISPY_ALLOWED_TEMPLATE_PACKS(self):
+        if self.RTD_EXT_THEME_ENABLED:
+            return ('semantic-ui',)
+        return None
```

Comment:
Looks like this should be returning an empty iterable or the defaults `("bootstrap", "uni_form", "bootstrap3", "bootstrap4")`